Doc/library/subprocess.rst

```diff
@@ -1292,11 +1292,17 @@ Replacing :func:`os.system`
 
    sts = os.system("mycmd" + " myarg")
    # becomes
-   sts = call("mycmd" + " myarg", shell=True)
+   retcode = call("mycmd" + " myarg", shell=True)
 
 Notes:
 
 * Calling the program through the shell is usually not required.
+* The :func:`call` return value is encoded differently to that of
+  :func:`os.system`.
+
+* The :func:`os.system` function ignores SIGINT and SIGQUIT signals while
```

Comment:
Excellent. :+1:  The bug report used the term "blocked", the correct behavior is "ignored". 
Ref - https://man7.org/linux/man-pages/man3/system.3.html